demo/molecule/buttonGroup/demo/index.scss

```diff
@@ -4,6 +4,14 @@
 
   &-section {
     margin: 10px;
+    
+    &--fullWidth {
+      border: 2px dashed blue;
```

Comment:
MAke use this border is not relative to buttons border values

Reply:
this is just some border style for the demo